model/status_board/v1/service_type.model

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Definition of a Status Board Service.
+class Service {
+  // Object creation timestamp.
+  CreatedAt Date
+
+  // Object modification timestamp.
+  UpdatedAt Date
+
+  // Object deletion timestamp.
+  DeletedAt Date
+
+  // The Application associated with the Service
+  ApplicationId String
+
+  // The name of the Service
+  Name String
+
+  ServiceEndpoint String
+
+  StatusType String
```

Comment:
This looks like an enumerated type. What are the possible values? Consider using an `enum` to better document it.

Reply:
It's a text field in our schema, it's not normalized.